src/client/app/model/card.ts

```diff
@@ -1,15 +1,26 @@
-import { Point } from './geometry';
+import { Point, Size } from './geometry';
 import { Arrow } from './arrow';
 import { CollaborativeObjectModel } from './collaborative-object-model';
+import CollaborativeList = gapi.drive.realtime.CollaborativeList;
 
 export class Card extends CollaborativeObjectModel {
   protected static modelName = 'Card';
+  readonly size: Size = {width:120, height:160};
   // Collaborative fields
   position: Point;
   text: string;
   selected: boolean;
-  incomingArrow: Arrow;
-  outgoingArrow: Arrow;
+  // TODO(eyuelt): Change these to normal arrays since CollaborativeLists
+  // inside custom collaborative objects are not actually collaborative.
```

Comment:
Imo, we should do this now. Seems harder to nest CollaborativeLists within a custom class (worse API compared to straight-up arrays), and potentially misleading.

Would only cost you a quick helper function to repeal/replace https://developers.google.com/google-apps/realtime/reference/gapi.drive.realtime.CollaborativeList.html#removeValue

Reply:
"repeal/replace" 😂

I think you might've sent me the wrong link. But I don't want to just use a helper to unwrap it here and still have all of the uses in the rest of the codebase pretend it's still a collaborative list. I want to change it everywhere. Since this would require multiple changes across the codebase, which would make merging my upcoming PR annoying, I'd rather do this in a separate PR.